File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/RapportoContabile.cs
using AgronicaCoreModelsSTD.baseClass;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class RapportoContabile : BaseCodeDescr
    {

        public bool cliente { get; set; }
        public bool fornitore { get; set; }
        public bool dipendente { get; set; }
        public bool terzista { get; set; }
        public bool legale { get; set; }
        public bool agente { get; set; }
        public bool consulente { get; set; }
        public bool flag_cancellazione { get; set; }

        public RapportoContabile(int codice) : base(codice, "")
        {
            this.flag_cancellazione = false;
        }

        public RapportoContabile()
        {
            this.flag_cancellazione = false;
        }

    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/RapportoContabile.ts
import { BaseCodeDescr } from '../baseClass/baseCodeDescr';

export class RapportoContabile extends BaseCodeDescr {
  cliente: boolean;
  fornitore: boolean;
  dipendente: boolean;
  terzista: boolean;
  legale: boolean;
  agente: boolean;
  consulente: boolean;
  flag_cancellazione: boolean;

  constructor(codice: number, description: string = '') {
    super(codice, description);
    this.cliente = false;
    this.fornitore = false;
    this.dipendente = false;
    this.terzista = false;
    this.legale = false;
    this.agente = false;
    this.consulente = false;
    this.flag_cancellazione = false;
  }
}
